Dockerfile

```
@@ -0,0 +1,10 @@
+FROM node:latest
+WORKDIR /app
+COPY package*.json ./
+RUN npm install
+RUN npm install -g nodemon
+COPY . .
+EXPOSE 3000 3001 3002
+USER root
```

Hadolint

[warning] Last USER should not be root

(DL3002)